packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -4,218 +4,230 @@ title: OAuth 2.0 and OpenID Connect Overview
 
 The high-level flow looks the same for both OpenID Connect and regular OAuth 2.0 flows. The primary difference is that an OpenID Connect flow results in an ID token, in addition to any access or refresh tokens.
 
-## Choosing an OAuth 2.0 flow
+## Choose an OAuth 2.0 flow
+
+The OAuth flow that you use depends on your use case. The following sections recommend OAuth 2.0 flows based on:
 
-Which OAuth flow that you use depends on your use case. The table below maps application types to our recommended OAuth 2.0 flows. If you'd like more information, keep reading for help with choosing an OAuth flow based on (1) the type of token that you need, and/or (2) the type of client application that you are building.
+* The type of application that you're building and the token types that the application requires
+* The type of client that you're building
 
-### Recommended flow by application type
+### What type of application are you building?
 
-The table shows you which OAuth 2.0 flow to use for the type of application that you are building.
+The following table shows you which OAuth 2.0 flow to use for the type of application that you're building.
 
-| Type of Application     | OAuth 2.0 flow / grant type                      | Access Token?    | ID Token?     |
+| Type of application     | OAuth 2.0 flow / grant type                      | Access token?    | ID token?     |
 | ----------------------- | ------------------------------------------------ | ---------------- | ------------- |
-| Server-side (aka Web), <br><br> Single-Page Application, <br><br> or Native | [Authorization Code with PKCE](#authorization-code-flow-with-pkce) or [Interaction Code](#interaction-code-flow) (Identity Engine only).                | &#9989;          | &#9989;      |
-| Trusted                 | [Resource Owner Password](#resource-owner-password-flow)            | &#9989;          | &#9989;      |
+| Server-side (aka web), <br />Single-Page Application, <br />or Native | [Authorization Code with PKCE](#authorization-code-flow-with-pkce) or<br> [Interaction Code](#interaction-code-flow) (Identity Engine only).                | &#9989;          | &#9989;      |
+| Trusted                 | [Interaction Code](#interaction-code-flow)            | &#9989;          | &#9989;      |
 | Service                 | [Client Credentials](#client-credentials-flow)                  | &#9989;          | &#10060;     |
 
-> **Note**: There is also an OAuth 2.0 [SAML 2.0 Assertion flow](#saml-2-0-assertion-flow), intended for a client app that wants to use an existing trust relationship without a direct user approval step at the authorization server. This supports access and ID tokens.
+> **Note**: The OAuth 2.0 [SAML 2.0 Assertion flow](#saml-20-assertion-flow) is also available. This flow is intended for client applications that want to use an existing trust relationship without a direct user approval step at the authorization server. It supports access and ID tokens.
```

Comment:
I think the "There's also a SAML 2.0 assertion flow." seems like a more natural phrasing.

Reply:
Sure - I'll buy that.